### ISSUE TITLE
provide a dvorak starting layer

### ISSUE THREAD

**Issue by bjornicus** [keyboardio/Chrysalis#362]
As noted in #268 having a way to share layers would be great, but in the shorter term just having a starting json used for import/export) for the common alternate layouts (e.g the same layouts as https://github.com/keyboardio/Kaleidoscope/wiki/Common-Alternate-Layouts) would be really useful.  I can probably do the dvorak one and submit a  PR with the layer 0 if you give some guidance on a good place to add it.  Perhaps a top level "sample_layouts" folder?

**Comment by bjornicus** [comment]
hmm, except I can't seem to copy the json it's showing me when I press import/export....

**Comment by Penthious** [comment]
@bjornicus This is what I use for dvorak. I would also be willing to set up something where at the click of a button it changes all the keys to dvorak or any other layout. Would just need some help with where I should start.

```
{
  "keymap": [
    {
      "keyCode": 65535,
      "label": "",
      "verbose": "Transparent"
    },
    {
      "keyCode": 30,
      "label": "1"
    },
    {
      "keyCode": 31,
      "label": "2"
    },
    {
      "keyCode": 32,
      "label": "3"
    },
    {
      "keyCode": 33,
      "label": "4"
    },
    {
      "keyCode": 34,
      "label": "5"
    },
    {
      "keyCode": 17152,
      "label": "Next",
      "extraLabel": "LEDEff."
    },
    {
      "keyCode": 224,
      "label": "LCtrl",
      "verbose": "Left Control"
    },
    {
      "keyCode": 228,
      "label": "RCtrl",
      "verbose": "Right Control"
    },
    {
      "keyCode": 24577,
      "label": "#1",
      "extraLabel": "Macro"
    },
    {
      "keyCode": 35,
      "label": "6"
    },
    {
      "keyCode": 36,
      "label": "7"
    },
    {
      "keyCode": 37,
      "label": "8"
    },
    {
      "keyCode": 38,
      "label": "9"
    },
    {
      "keyCode": 39,
      "label": "0"
    },
    {
      "keyCode": 17409,
      "label": "1",
      "extraLabel": "LockTo"
    },
    {
      "keyCode": 53,
      "label": "`"
    },
    {
      "keyCode": 52,
      "label": "'"
    },
    {
      "keyCode": 54,
      "label": ","
    },
    {
      "keyCode": 55,
      "label": "."
    },
    {
      "keyCode": 19,
      "label": "P"
    },
    {
      "keyCode": 28,
      "label": "Y"
    },
    {
      "keyCode": 43,
      "label": "Tab"
    },
    {
      "keyCode": 42,
      "label": "BSpc",
      "verbose": "Backspace"
    },
    {
      "keyCode": 44,
      "label": "Space"
    },
    {
      "keyCode": 40,
      "label": "Enter"
    },
    {
      "keyCode": 9,
      "label": "F"
    },
    {
      "keyCode": 10,
      "label": "G"
    },
    {
      "keyCode": 6,
      "label": "C"
    },
    {
      "keyCode": 21,
      "label": "R"
    },
    {
      "keyCode": 15,
      "label": "L"
    },
    {
      "keyCode": 56,
      "label": "/"
    },
    {
      "keyCode": 75,
      "label": "PgUp",
      "verbose": "Page Up"
    },
    {
      "keyCode": 4,
      "label": "A"
    },
    {
      "keyCode": 18,
      "label": "O"
    },
    {
      "keyCode": 8,
      "label": "E"
    },
    {
      "keyCode": 24,
      "label": "U"
    },
    {
      "keyCode": 12,
      "label": "I"
    },
    {
      "keyCode": 41,
      "label": "Esc"
    },
    {
      "keyCode": 227,
      "label": "LGui",
      "verbose": "Left Gui"
    },
    {
      "keyCode": 226,
      "label": "LAlt",
      "verbose": "Left Alt"
    },
    {
      "keyCode": 230,
      "label": "RAlt",
      "verbose": "AltGr"
    },
    {
      "keyCode": 7,
      "label": "D"
    },
    {
      "keyCode": 11,
      "label": "H"
    },
    {
      "keyCode": 23,
      "label": "T"
    },
    {
      "keyCode": 17,
      "label": "N"
    },
    {
      "keyCode": 22,
      "label": "S"
    },
    {
      "keyCode": 45,
      "label": "-"
    },
    {
      "keyCode": 226,
      "label": "LAlt",
      "verbose": "Left Alt"
    },
    {
      "keyCode": 51,
      "label": ";"
    },
    {
      "keyCode": 20,
      "label": "Q"
    },
    {
      "keyCode": 13,
      "label": "J"
    },
    {
      "keyCode": 14,
      "label": "K"
    },
    {
      "keyCode": 27,
      "label": "X"
    },
    {
      "keyCode": 17452,
      "label": "2",
      "extraLabel": "ShiftTo"
    },
    {
      "keyCode": 225,
      "label": "LShift",
      "verbose": "Left Shift"
    },
    {
      "keyCode": 229,
      "label": "RShift",
      "verbose": "Right Shift"
    },
    {
      "keyCode": 17452,
      "label": "2",
      "extraLabel": "ShiftTo"
    },
    {
      "keyCode": 5,
      "label": "B"
    },
    {
      "keyCode": 16,
      "label": "M"
    },
    {
      "keyCode": 26,
      "label": "W"
    },
    {
      "keyCode": 25,
      "label": "V"
    },
    {
      "keyCode": 29,
      "label": "Z"
    },
    {
      "keyCode": 46,
      "label": "="
    }
  ],
  "colormap": [
    [
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15
    ],
    [
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15
    ],
    [
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15
    ],
    [
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15
    ],
    [
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15,
      15
    ]
  ],
  "palette": []
}
```

**Comment by TreTuna** [comment]
I made a quick PR that adds buttons for all the default layouts. https://github.com/keyboardio/Chrysalis/pull/366

**Comment by TreTuna** [comment]
Solved in #366
